src/ui/components/VPNExpandableAppList.qml

```diff
@@ -0,0 +1,143 @@
+    clip: true
+    canGrowVertical: true
+    accessibleName: name
+    states: [
+        State {
+            name: "list-hidden"
+            PropertyChanges {
+                target: toggleArrow
+                rotation: -90
+
+            }
+            PropertyChanges {
+                target: appRow
+                height: appRowHeader.height
+            }
+
+        },
+        State {
+            name: "list-visible"
+
+            PropertyChanges {
+                target: toggleArrow
+                rotation: 0
+
+            }
+            PropertyChanges {
+                target: appRow
+                height: appRowHeader.height +applist.contentHeight + applist.anchors.topMargin
+            }
+
+        }
+    ]
+
+
+    RowLayout {
+        id: appRowHeader
+        anchors.top: parent.top
+        anchors.left: parent.left
+        width: appRow.width
+        spacing: 0
+        VPNServerListToggle {
+            id: toggleArrow
+            Layout.leftMargin: 15 - enabledAppRow.anchors.leftMargin
+            Layout.rightMargin: 15
+        }
+        ColumnLayout {
+            Layout.alignment: Qt.AlignLeft
+            Layout.fillWidth: true
+            spacing: 0
+
+            RowLayout {
+                Layout.alignment: Qt.AlignLeft
+                Layout.fillWidth: true
+
+                VPNInterLabel {
+                    id: label
+                    text: header
+                    Accessible.role: Accessible.Heading
+                    color: Theme.fontColorDark
+                    horizontalAlignment: Text.AlignLeft
+                }
+
+                VPNTextBlock{
+                    visible: !listVisible || applist.count == 0 || !isEnabled
+                    text: " (%0)".arg(applist.count)
+                }
+            }
+
+            VPNTextBlock {
+                id: enabledAppSubtext
+                text: description
+                Layout.fillWidth: true
+            }
+        }
+        VPNLinkButton{
+            Layout.alignment: Qt.AlignRight
+            // (Un)protect-All Button
+            labelText: actionText
+            onClicked: {onAction()}
+            enabled: isEnabled
+            opacity: isEnabled ? 1: 0.5
+        }
+    }
+
+    VPNList {
+        id: applist
+        model: listModel
+
+        anchors.top: appRowHeader.bottom
+        anchors.topMargin: count > 0 ? 16 : 0
+        anchors.left: appRowHeader.left
+        width: appRow.width
+        height: contentItem.childrenRect.height
+        spacing: 26
+        listName: header
+
+        visible: listVisible && count > 0
+
+        delegate: VPNCheckBoxRow {
+            labelText: appName
+            subLabelText: appID
+            isChecked: appIsEnabled
+            isEnabled: appRow.isEnabled
+            showDivider:false
+            onClicked: VPNAppPermissions.flip(appID)
+            visible: true
+            width: applist.width
+            anchors.left: applist.left
```

Comment:
`qrc:/ui/components/VPNExpandableAppList.qml:128:19: QML VPNCheckBoxRow: Cannot anchor to an item that isn't a parent or sibling. qrc:/ui/components/VPNExpandableAppList.qml: 128`

A strange ListView gotcha: the parent these delegates is actually `applist.contentItem`, not `applist` itself.  This just needs to be `anchors.left: parent.left`.